src/Appium.Net/Appium/Android/Interfaces/IPushesFiles.cs

```diff
@@ -24,7 +24,7 @@ public interface IPushesFiles : IInteractsWithFiles
         /// </summary>
         /// <param name="pathOnDevice">Path to file to write data to on remote device</param>
         /// <param name="stringData">A string to write to remote device</param>
-        void PushFile(string pathOnDevice, string base64Data);
+        void PushFile(string pathOnDevice, string stringData);
```

Comment:
I'm confused here. Which data type is actually expected by this API? the appium server API does expect a base64-encoded string.

Reply:
There is indeed an overload that also accepts base64-encoded string.
https://github.com/appium/dotnet-client/blob/44c2c314372290e4a0d3397eb887349c0ec10f49/src/Appium.Net/Appium/Android/Interfaces/IPushesFiles.cs#L22-L34

Reply:
I just wanted to make sure the argument that is actually sent to the server is base64-encoded.